pipenv/vendor/background.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import sys
+import multiprocessing
+
+if sys.version_info.major < 3:
```

Comment:
concurrent.futures is built in on 3.4+ i believe.

Reply:
3.4 actually!  removed the vendorized version for 3